<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Battle</class>
 <widget class="QWidget" name="Battle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>40</y>
     <width>205</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>38</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="oneSpeed">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="twoSpeed">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>1161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,6,5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
        <item>
         <widget class="QProgressBar" name="hpPerA">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="value">
           <number>24</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hpA">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1">
        <item>
         <widget class="QProgressBar" name="hpPerB">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="value">
           <number>24</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hpB">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="picA">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>550</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>289</width>
     <height>271</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="picB">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>550</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>289</width>
     <height>271</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="back1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光行书_CNKI</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/back/background/back1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="skill1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>490</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光黑体_CNKI</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#e68200</string>
   </property>
   <property name="text">
    <string>水炮</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="back2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>440</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光行书_CNKI</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/back/background/back1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="skill2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>500</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光黑体_CNKI</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>水炮</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>210</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,1,3,1">
    <item>
     <widget class="QLabel" name="enemybuff1">
      <property name="minimumSize">
       <size>
        <width>79</width>
        <height>39</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>79</width>
        <height>39</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#00EE76</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="enemybuff1N">
      <property name="minimumSize">
       <size>
        <width>26</width>
        <height>39</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>26</width>
        <height>39</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#00EE76</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="enemybuff2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>39</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>39</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FF6347</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="enemybuff2N">
      <property name="minimumSize">
       <size>
        <width>26</width>
        <height>39</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>26</width>
        <height>39</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FF6347</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="3,1,3,1">
    <item>
     <widget class="QLabel" name="minebuff1">
      <property name="minimumSize">
       <size>
        <width>79</width>
        <height>39</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>79</width>
        <height>39</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#00EE76</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="minebuff1N">
      <property name="minimumSize">
       <size>
        <width>26</width>
        <height>39</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>26</width>
        <height>39</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#00EE76</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="minebuff2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>39</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>39</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FF6347</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="minebuff2N">
      <property name="minimumSize">
       <size>
        <width>26</width>
        <height>39</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>26</width>
        <height>39</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FF6347</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
